### PR TITLE
[17.0][MIG] l10n_ir_accounting: Migration to 17.0

### PR DESCRIPTION
**Pull request by hamidk1010**

### CODE REVIEW

**Review by dreispt — APPROVED** [review]

File: l10n_ir_account/data/template/account.account-ir.csv [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+"l10n_ir_chart_621310","سهم جاری هزینه تاسیس وقبل از بهره برداری","621310","expense","","0"
+"l10n_ir_chart_621311","حق الزحمه بازرسین","621311","expense","","0"
+"l10n_ir_chart_621312","حق حضور اعضاء هیئت مدیره","621312","expense","","0"
+"l10n_ir_chart_621319","سایر هزینه های اداری و تشکیلاتی","621319","expense","","0"
+"l10n_ir_chart_621501","سود و زیان ناشی از تسعیر ارز","621501","expense","","0"
+"l10n_ir_chart_621502","سود و زیان ناشی از فروش دارائیها","621502","expense","","0"
+"l10n_ir_chart_621503","سود و زیان ناشی از بیمه دارائیها","621503","expense","","0"
+"l10n_ir_chart_621504","سود سپرده","621504","expense","","0"
+"l10n_ir_chart_621505","درآمدهای متفرقه","621505","expense","","0"
+"l10n_ir_chart_621506","هزینه های باشگاه فرهنگی ورزشی کاشی باستان","621506","expense","","0"
```
Comment (saeed-raeisi) [inline l10n_ir_account/data/template/account.account-ir.csv:351]: Hello
This coa needs to be modified
These items are related to my previous module and I was supposed to correct them in the new version
"l10n_ir_chart_621506","هزینه های باشگاه فرهنگی ورزشی کاشی باستان","621506","expense","","0"
جدول حساب مختص صعنت کاشی - و کارخانه باستان می باشد.

File: l10n_ir_account/data/template/account.account-ir.csv [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+"l10n_ir_chart_811301","انحراف هزینه دستمزد","811301","expense","","0"
+"l10n_ir_chart_811302","انحراف هزینه سربار","811302","expense","","0"
+"l10n_ir_chart_811401","انتقال هزینه های دستمزد","811401","expense","","0"
+"l10n_ir_chart_811402","انتقال هزینه های سربار","811402","expense","","0"
+"l10n_ir_chart_911001","ضمانت نامه های انجام مناقصه ومزایده","911001","off_balance","","0"
+"l10n_ir_chart_911002","ضمانت نامه های حسن انجام کار","911002","off_balance","","0"
+"l10n_ir_chart_911003","اسناد تضمینی دریافتی از کارکنان","911003","off_balance","","0"
+"l10n_ir_chart_911004","ضمانت نامه های بانکی","911004","off_balance","","0"
+"l10n_ir_chart_911005","سایر اسناد تضمینی به نفع شرکت","911005","off_balance","","0"
+"l10n_ir_chart_911006","اسناد تضمینی دریافتی از نمایندگان کاشی","911006","off_balance","","0"
```
Comment (saeed-raeisi) [inline l10n_ir_account/data/template/account.account-ir.csv:373]: same here

File: l10n_ir_account/data/template/account.account-ir.csv [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+"l10n_ir_chart_911008","اسناد تضمینی تنخواه","911008","off_balance","","0"
+"l10n_ir_chart_911009","اسناد تضمینی دریافتی از نمایندگان کاشی(جدید)","911009","off_balance","","0"
```
Comment (saeed-raeisi) [inline l10n_ir_account/data/template/account.account-ir.csv:375]: +

File: l10n_ir_account/data/template/account.account-ir.csv (hunk 1 above)
Comment (hamidk1010) [reply l10n_ir_account/data/template/account.account-ir.csv:351]: hi, all the coa  will be  refine  in a other pull request, after merging this

Comment (hamidk1010) [reply l10n_ir_account/data/template/account.account-ir.csv:351]: also i think we could have two coa: **new** and **old**
[old coa.pdf](https://github.com/OCA/l10n-iran/files/13543165/old.coa.pdf)
[new coa.pdf](https://github.com/OCA/l10n-iran/files/13543169/new.coa.pdf)

**Comment by hamidk1010** [comment]
@dreispt  what is the problem in merging?

**Comment by dreispt** [comment]
Let me try:

**Comment by dreispt** [comment]
@sbidoul Can you help? I checked dotfiles, and they seem OK to me. Not sure what the problem is.
Thanks!

File: l10n_ir_account/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Iran - Accounting",
+    "version": "1.0.0",
```
Comment (sbidoul) [inline l10n_ir_account/__manifest__.py]:
```suggestion
    "version": "17.0.1.0.0",
```

@saeed-raeisi @dreispt this is the cause of the merge error. We'll need to investigate why pylint-odoo did not flag this error, and why the bot did not report the exception on the PR.

**Pull request merged**